build-scripts/xc-universal-binary.sh

```diff
@@ -49,7 +49,7 @@ for i in "${!LIBS_ARCHS[@]}"; do
         OPENSSL_DIR=$LIBSDIR/ios/$LIB_ARCH/openssl \
         SQLCIPHER_LIB_DIR=$LIBSDIR/ios/$LIB_ARCH/sqlcipher/lib \
         SQLCIPHER_INCLUDE_DIR=$LIBSDIR/ios/$LIB_ARCH/sqlcipher/include \
-    $HOME/.cargo/bin/cargo build -p $FFI_TARGET --lib $RELFLAG --target ${IOS_TRIPLES[$i]}
+    $HOME/.cargo/bin/cargo build --locked -p $FFI_TARGET --lib $RELFLAG --target ${IOS_TRIPLES[$i]}
```

Comment:
@ncalexan how would I go about passing this `--locked` flag through to cargo on android builds?

Reply:
We accommodate flags in the environment already: see https://github.com/ncalexan/rust-android-gradle#passing-arguments-to-cargo.  But I'm not sure that this option will be accommodated with this approach.  I generally didn't want to open the command line up as a free-for-all, but here we are.

I'd take a patch to https://github.com/mozilla/rust-android-gradle/ to:
- add a `locked` variable, like https://github.com/ncalexan/rust-android-gradle/blob/master/plugin/src/main/kotlin/com/nishtahir/CargoBuildTask.kt#L69; or
- add `cargoBuildArguments` explicitly to the configuration block and feed them in.